src/main/java/user/postreview/PostReview.java

```diff
@@ -14,8 +19,7 @@ public PostReview(IReviewDAO reviewDAO, IInternshipDAO internshipDAO) {
         this.internshipDAO = internshipDAO;
```

Comment:
Is the internshipDAO still in use? It seems like the reviewDAO is doing the job instead, since all the internshipDAO below has been commented out. Also should these commented code be deleted? 
I know you didn't write these code, but maybe confirm that and make necessary changes?

Reply:
I already asked them about commented out code but it seems like they're still working out things to implement and things to delete. Either way, when posting a review the internship database should be used at some point since it has to be updated with the new review that was posted being put into its ArrayList of reviews.